lib/gnat/jetstream/api/kv.ex

```diff
@@ -265,6 +265,28 @@ defmodule Gnat.Jetstream.API.KV do
     end
   end
 
+  @doc """
+  Returns a list of all the buckets in the KV
+  """
+  @spec list_buckets(conn :: Gnat.t()) :: {:error, term()} | {:ok, list(String.t())}
+  def list_buckets(conn) do
+    case Stream.list(conn) do
```

Comment:
I would swap this `case` statement for a `with` since it has an obvious happy-path pattern.

`with {:ok, %{streams: streams}} <- Stream.list(conn) do`